src/pycatcher/catch.py

```diff
@@ -14,31 +15,101 @@
 from scipy.special import inv_boxcox
 
 
-# Configure logging
-logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
-logger = logging.getLogger(__name__)
+# Configure logging based on environment with more detailed format
```

Comment:
This function enables the log visibility as needed. The default level is WARNING if the user doesn't set it.